packages/init/src/tasks/modifyIOS.ts

```diff
@@ -0,0 +1,96 @@
+import fs from 'fs';
+import path from 'path';
+
+import logger from '../utils/logger.js';
+
+function findProjectPbxProjPath(cwd: string) {
+  const iosPath = path.join(cwd, 'ios');
+  const iosDir = fs.readdirSync(iosPath);
+  const xcodeProjDir = iosDir.find((dir) => dir.endsWith('.xcodeproj'));
+
+  if (!xcodeProjDir) {
+    throw new Error('No directory with "project.pbxproj" found');
+  }
+
+  const pbxProjPath = path.join(iosPath, xcodeProjDir, 'project.pbxproj');
+  return pbxProjPath;
+}
+
+function modifyPbxProjConfig(config: string): string {
+  const fallbackMessage =
+    'To finish the setup for iOS, You need to edit the file manually.\n' +
+    'Follow instructions at: https://re-pack.netlify.app/docs/getting-started/#4-configure-xcode';
+
+  const pbxShellScriptBuildPhaseIndex = config.indexOf(
+    'PBXShellScriptBuildPhase'
+  );
+
+  if (pbxShellScriptBuildPhaseIndex === -1) {
+    throw new Error(
+      'Could not find PBXShellScriptBuildPhase section in project.pbxproj\n' +
+        fallbackMessage
+    );
+  }
+
+  const shellScriptIndex = config.indexOf(
+    'shellScript =',
```

Comment:
Are you sure the `Bundle React Native code and images` in which we must add the bundle command will always be the first in the `PBXShellScriptBuildPhase` section?

Reply:
Wouldn't looking for `react-native-xcode.sh` and inserting `export BUNDLE_COMMAND=webpack-bundle` in a line above be safer?

Reply:
Great point about the order in that section! I'll make sure to include that. 

Regarding the 2nd comment - I would prefer to stick to the project itself instead of modifying something that comes from `node_modules`, which could potentially get overwritten with each `yarn install` in the project

Reply:
Rewrote the whole modifyIOS part using `xcode` package for modyfing `project.pbxproj` files